content/docs/command-reference/remote/modify.md

```diff
@@ -624,12 +624,10 @@ more details.
 
 > Please note our [Privacy Policy (Google APIs)](/doc/user-guide/privacy).
 
-- `gdrive_acknowledge_abuse` - acknowledge the risk of downloading potential
-  malware. Files identified as
-  [abusive](https://support.google.com/docs/answer/148505) (malware, etc.) are
-  only downloadable by the owner. Consider using this option if you see a
-  `This file has been identified as malware or spam and cannot be downloaded`
-  error.
+- `gdrive_acknowledge_abuse` - acknowledge the risk of downloading potentially
+  [abusive](https://support.google.com/docs/answer/148505). Files identified
```

Comment:
potentially abusive ... ? (files, content, something is needed here)

Reply:
let's keep this part

"Consider using this option if you see a `This file has been identified as malware or spam and cannot be downloaded`  error."
 
 it's important for search, etc

Reply:
CC @jorgeorpinel

Reply:
Answered in #3795